packages/fast-components-styles-msft/src/context-menu-item/index.ts

```diff
@@ -13,52 +10,62 @@ import {
     neutralForegroundHover,
     neutralForegroundRest,
 } from "../utilities/color";
-import { applyFocusVisible } from "@microsoft/fast-jss-utilities";
-import { applyTypeRampConfig } from "../utilities/typography";
-import { toPx } from "@microsoft/fast-jss-utilities";
+import { applyFocusVisible, toPx } from "@microsoft/fast-jss-utilities";
+import { applyCornerRadius, applyFocusPlaceholderBorder } from "../utilities/border";
+import { applyDisabledState } from "../utilities/disabled";
+import { scaleApplyTypeRampConfigWithDensity } from "../utilities/typography";
 
-const styles: ComponentStyles<ContextMenuItemClassNameContract, DesignSystem> = {
-    contextMenuItem: {
-        listStyleType: "none",
-        height: density(32),
-        display: "grid",
-        boxSizing: "border-box",
-        gridTemplateColumns: "40px auto 40px",
-        gridTemplateRows: "auto",
-        alignItems: "center",
-        padding: "0",
-        margin: "0 4px",
-        color: neutralForegroundRest,
-        whiteSpace: "nowrap",
-        overflow: "hidden",
-        cursor: "default",
-        transition: "all 0.2s ease-in-out",
-        ...applyTypeRampConfig("t7"),
-        borderRadius: (config: DesignSystem): string => {
-            const designSystem: DesignSystem = withDesignSystemDefaults(config);
+const styles: ComponentStyles<ContextMenuItemClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<ContextMenuItemClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const padding: number =
+        horizontalSpacingNumber(-2)(designSystem) +
+        16 +
+        horizontalSpacingNumber()(designSystem);
 
-            return toPx(designSystem.cornerRadius);
+    return {
+        contextMenuItem: {
+            listStyleType: "none",
+            boxSizing: "border-box",
+            height: height(),
+            display: "grid",
+            gridTemplateColumns: `${toPx(padding)} auto ${toPx(padding)}`,
+            gridTemplateRows: "auto",
+            justifyItems: "center",
+            alignItems: "center",
+            padding: "0",
+            margin: "0 4px",

```

Comment:
should this be `designSystem.designUnit` similar to the translate for CTA? 

What about horizontalSpacing utility? This technically seems to be providing horizontal spacing, is there a reason why it is not affected by density?

Reply:
Yes, I think it is designUnit. I've  updated this and a few other locations. With horizontalSpacing we're adjusting the number of designUnits with density, not the value. In the case of 4px I think it will always be only one unit as it's intended to be small, essentially just a little ridge margin in this and many other cases.